### PR TITLE
[TW-1544] Monitors overview refresh

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Updated the featured snippet
* Reorganized and relocated content as needed
* Added links to and brief intros about majors topics in the section

### CODE REVIEW

**Review by akinard-postman — APPROVED** [review]
Looks good! 👍

File: src/pages/docs/monitoring-your-api/intro-monitors.md [hunk 1]
```diff
@@ -38,30 +38,33 @@ contextual_links:
...
+Depending on your [plan](https://www.postman.com/pricing), you can [configure monitors to run from static IPs](/docs/monitoring-your-api/using-static-IPs-to-monitor/). This enables you to monitor APIs that are behind a restricted firewall.
 
-Get alerted by email if a test fails or errors occur, or [set up integrations](/docs/integrations/intro-integrations/) to be notified over Slack and other channels. All results are recorded on the monitor’s dashboard, so you can view past results or inspect trends over time.
+You can [view results for collection-based monitors](/docs/monitoring-your-api/viewing-monitor-results/) to learn more about the monitors you're running. You can also [manage your team's monitor usage](/docs/monitoring-your-api/monitor-usage/), enabling you to track your team's resource usage and purchase more blocks of monitoring calls.
 
-Learn how to [set up a collection-based monitor](/docs/monitoring-your-api/setting-up-monitor/).
+> If you experience a problem with collection-based monitors, you can [troubleshoot monitors in Postman](/docs/monitoring-your-api/troubleshooting-monitors/).
+
+## Monitor FAQs
```
Comment (mark-anthony-postman) [inline src/pages/docs/monitoring-your-api/intro-monitors.md:61]: Maybe put this section after Monitor use cases? Feels like it should be the last thing on the page

Comment (MatthewPinsker-postman) [reply src/pages/docs/monitoring-your-api/intro-monitors.md]: Yes, I agree. This is now updated. Thanks!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**